assets/src/apps/authoring/components/ComponentToolbar/AddComponentToolbar.tsx

```diff
@@ -75,14 +82,14 @@ const AddComponentToolbar: React.FC<{
       const PartClass = customElements.get(partComponent.authoring_element);
       if (PartClass) {
         // only ever add to the current activity, not a layer
-
+        setNewPartAddCounter(newPartAddCounter + 1);
         const part = new PartClass() as any;
         const newPartData = {
           id: `${partComponentType}-${guid()}`,
           type: partComponent.delivery_element,
           custom: {
-            x: 10,
-            y: 10,
+            x: 10 * newPartAddCounter,
```

Comment:
These changes are done to handle - "`As new components are added, offset the location these are placed by 10 px. So the first component should be at (10,10) by default, the second component should default to (20,20), then (30,30), and so on`"